limits-requests/config.yaml

```diff
@@ -0,0 +1,14 @@
+code:
+  name: dkube-examples
+  url: https://github.com/oneconvergence/dkube-examples.git
+  branch: tensorflow
+  private: false
+  
+arguments:
+  code: dkube-examples
+  run_script: stress --vm-bytes 10M
```

Comment:
to test limit requests work, the example needs to allocate more memory than the limit. Lets add a key at the end , "status: failed". Robot automation should check this key and expect pipeline to fail and not fail the test case

Reply:
added the status field, verified pipeline failed and robot checked status and it passed.

Reply:
allocated more memory and cpu, dkube giving error insufficient cpu and memory